src/ExceptionRenderer/RendererAbstract.php

```diff
@@ -134,7 +134,7 @@ protected function getStackTrace(bool $shorten): array
 
         $both_atk = $this->exception instanceof Exception && $this->parent_exception instanceof Exception;
         $c = min(count($trace), count($parent_trace));
-        for ($i = 0; $i < $c; $i++) {
+        for ($i = 0; $i < $c; ++$i) {
```

Comment:
is `++$i` somehow better than `$i++` here?

Reply:
I do not like it too, but for some reasons it is prefered by CS fixed by its rules - so if you agree, I can add one extra CS rule to change it to the original form.

Reply:
Described here: https://stackoverflow.com/a/1756021/1466341
At least 10 years ago they claim that pre-increment was 10% faster than post-increment.

Reply:
Tested: `++$i` is always faster by 10-25% than `$i++` so let's stick with `++$i` where possible.